source/install/use-kubernetes-mattermost.rst

```diff
@@ -0,0 +1,89 @@
+.. _use-kubernetes-mattermost:
+
+Using Mattermost Operator Functionality
+======================================
+
+Rolling Upgrades
+----------------
+
+The Mattermost Kubernetes Operator supports rolling upgrades so you can upgrade
+your Mattermost deployment with zero downtime. This process
+requires at least two replicas as a rolling upgrade cannot be performed if there is only one pod.
+Replicas are created when a user count is selected and exceeds 100.
```

Comment:
Does this happen automatically?

Reply:
Do you mean the rolling upgrades or the creation of the replicas?

Reply:
This line is clarifying that rolling upgrades are only possibly for deployments of more than 100 users right now since you need more than 1 replica